src/Compilers/CSharp/Test/Emit/Emit/EditAndContinue/EditAndContinueClosureTests.cs

```diff
@@ -3631,5 +3631,407 @@ .maxstack  9
   IL_0070:  ret
 }");
         }
+
+        [Fact]
+        public void TwoStructClosures()
+        {
+            var source0 = MarkedSource(@"
+public class C 
+    {
+    public void F()            
+    {
+        int x = 0;
+        {
+            int y = 0;
+            // Captures two struct closures
+            int L() => x + y;
+        }
+    }
+}");
+
+            var source1 = MarkedSource(@"
+public class C 
+    {
+    public void F()            
+    {
+        int x = 0;
+        {
+            int y = 0;
+            // Captures two struct closures
+            int L() => x + y + 1;
```

Comment:
>int L() => x + y + 1; [](start = 12, length = 21)

This is not testing what we want - you are missing <N> markers to let the compiler know what syntax nodes match

Reply:
Thanks!

---
In reply to: [137682130](https://github.com/dotnet/roslyn/pull/21971#discussion_r137682130) [](ancestors = 137682130)